pam/integration-tests/vhs-helpers_test.go

```diff
@@ -302,28 +313,50 @@ func (td tapeData) Output() string {
 	return txt
 }
 
-func (td tapeData) ExpectedOutput(t *testing.T, outputDir string) string {
+func checkDataRace(t *testing.T, raceLog string) {
 	t.Helper()
 
-	outPath := filepath.Join(outputDir, td.Output())
-	out, err := os.ReadFile(outPath)
-	require.NoError(t, err, "Could not read output file of tape %q (%s)", td.Name, outPath)
-	got := string(out)
+	if !testutils.IsRace() || raceLog == "" {
+		return
+	}
+
+	content, err := os.ReadFile(raceLog)
+	if err == nil || errors.Is(err, os.ErrNotExist) {
+		return
+	}
+	require.NoError(t, err, "TearDown: Error reading race log %q", raceLog)
+
+	out := string(content)
+	if strings.TrimSpace(out) == "" {
+		return
+	}
 
-	if testutils.IsRace() && strings.Contains(got, "WARNING: DATA RACE") &&
-		strings.Contains(got, "bubbles/cursor.(*Model).BlinkCmd.func1") {
+	if strings.Contains(out, "bubbles/cursor.(*Model).BlinkCmd.func1") {
```

Comment:
does each race logfile only contain one race? if not, we might ignore other detected races here if it happens in the same run as this well known race.

Reply:
No the log file contains all the races logs it detects AFAIK... Here i wanted to preserve the same behavior we had before.

Indeed, ideally we should parse it and still fail in case some other kind of race is detected, but I didn't want to go too far...

Reply:
> Here i wanted to preserve the same behavior we had before.

Fair enough, but then let's add a `// TODO` comment

Reply:
done.